src/pydata_sphinx_theme/assets/styles/sections/_header.scss

```diff
@@ -100,3 +109,25 @@
   padding-left: 2px;
   vertical-align: middle;
 }
+
+label.primary-toggle {
```

Comment:
The centering and font size seems slightly off:

![image](https://user-images.githubusercontent.com/1839645/174751279-27708e45-7c8b-477c-9121-37e325ca1c0a.png)

We should try to make it vertically centered, and maybe a slightly bigger font IMO

Reply:
[e835a9f](https://github.com/pydata/pydata-sphinx-theme/pull/726/commits/e835a9f8801c8ca45934883f7814f90bc0a4077b)